modules/salt/pages/yomi.adoc

```diff
@@ -4,24 +4,41 @@
 Yomi (yet one more installer) is an installer for {suse} and openSUSE operating systems.
 Yomi is designed as a SaltStack state, and can be used for installing {suse} operating systems on new systems.
 
-In the context of {productname} we can use Yomi as one step for the provisioning of a new system, and as an alternative to AutoYaST.
+In {productname}, Yomi can be used as part of provisioning new clients, or as an alternative to AutoYaST.
 
 Yomi consists of two components:
 
 * The Yomi formula, which contains the Salt states and modules required to perform the installation.
-* The LiveCD operating system image, which includes the `pre-configured ``salt-minion`` service.
+* The operating system image, which includes the `pre-configured ``salt-minion`` service.
 
-Both components can be used independently of {productname}, or integrated with it.  In this section we will discuss the last option, but for an independent installation, configuration and usage reference to the upstream documentation in https://github.com/openSUSE/yomi and the assets in the build repository: https://build.opensuse.org/project/show/systemsmanagement:yomi
+Both components can be used independently of {productname}, or integrated with it.
+This section describes how to use it with {productname}.
+For more information about using Yomi independently, see https://github.com/openSUSE/yomi.
+For build assets, see https://build.opensuse.org/project/show/systemsmanagement:yomi
 
-To use Yomi for installing your client operating system, you need to follow this process:
+To use Yomi for installing a client operating system, follow this process:
 
 * Install the Yomi formula package.
 * Prepare the Salt pillar for the new installation.
 * Boot the new client using the PXE boot image for Yomi.
 
+
+
 == Install the ``yomi-formula``
 
-The first required component is the ``yomi-formula``, already available as a package in {productname}.
+Before you begin, you need to install the ``yomi-formula``, which is available as a package in {productname}.
+
+The ``yomi-formula`` package contains the Salt states and modules that describe the Yomi state, the {productname} formulas that will allow the creation of the pillar using the {productname} {webui}, the documentation about the different sections of the pillar, and some examples about how to parameterize installations based on openSUSE, MicroOS or SLE.
+
+The formula package performs these actions:
+
+* Adds a new configuration file called ``yomi-formula.conf`` in the [path]``/etc/salt/master.d/`` directory.
```

Comment:
... for finding the Python module and states required by Yomi.

I would comment the reason of this configuration change.